plugins/outputs/elasticsearch/README.md

```diff
@@ -210,6 +210,16 @@ This plugin will format the events in the following way:
   ##               NaNs and inf will be replaced with the given number, -inf with the negative of that number
   # float_handling = "none"
   # float_replacement_value = 0.0
+  
+  ## Pipeline Config
+  ## To use a ingest pipeline, set this to the name of the pipeline you want to use.
+  # use_pipeline = "my_pipeline"
+  ## Additionally, you can specify a tag name using the notation {{tag_name}}
+  ## which will be used as part of the pipeline name. If the tag does not exist,
+  ## the default pipeline will be used as the pipeline. If no default pipeline is set,
+  ## no pipeline is used for the metric.
+  # use_pipeline = "{{es_pipeline}}"
```

Comment:
We usually only mention a variable once in the sample config. Could you merge the two sections describing "use_pipeline" into one and remove one of the commented settings? It's just a project readme style that makes all the various plugins look more uniform.

Also maybe move the description of default_pipeline to be right before that setting?
Thanks

Reply:
I was trying to follow the same format as the index name section. But this can be merged, i just wanted to make it clear between using a static value and a dynamic value

Reply:
@reimda are you good with this as-is?